docs/src/content/docs/it/guides/sidebar.mdx

```diff
@@ -461,19 +487,16 @@ La configurazione sopra genera la seguente barra laterale:
 <SidebarPreview
 	config={[
 		{
-			label: 'Guide',
+			label: 'Guides',
```

Comment:
The entire "Collapsing groups" ("Gruppi compressi") section which contains 3 examples is not really consistent regarding the translations of labels and autogenerated folders.

The approach we have been trying to use in translations for this specific section is to assume  that the example are based on a monolingual website in the language of the reader.

This means we usually try to:

- Translate labels (except for the autogenerated `advanced` folder)
- Use links referring to the reader language (`it` prefix in this case)

Would you mind doing a pass on these 3 examples of this section to add more consistency? If not, or this was not clear or have any question, definitely let me know.